src/views/waypoint/home/server.ts

```diff
@@ -0,0 +1,87 @@
+import slugify from 'slugify'
+// TODO: need to discuss from whence we should
+// TODO: source content down the road. For now,
+// TODO: sourcing from JSON for demo purposes.
+import TEMP_CONTENT from './content.json'
+
+const productName = 'Waypoint'
+const productSlug = 'waypoint'
+
+async function getStaticProps(): Promise<$TSFixMe> {
+  const CONTENT = TEMP_CONTENT as $TSFixMe
+
+  // TODO: Content blocks content should likely
+  // TODO: be fetched dynamically, currently things
+  // TODO: like the title and description of docs pages
+  // TODO: are hard-coded alongside the URL to link to.
+  // TODO: Should be possible to fetch the title and description
+  // TODO: at build time from said URL, to ensure the displayed
+  // TODO: content is accurate, while not needing to be
+  // TODO: manually kept up to date?
+  CONTENT.blocks = CONTENT.blocks.map((block) => {
+    const { type, heading } = block
+    if (type !== 'heading') return block
+    return {
+      ...block,
+      slug: slugify(heading, { lower: true }),
+    }
+  })
+
+  const navData = [
+    { title: 'Introduction', fullPath: '/waypoint/docs/intro' },
+    {
+      title: 'Getting Started',
+      fullPath: '/waypoint/docs/getting-started',
+    },
+    { divider: true },
+    { heading: 'Resources' },
+    {
+      title: 'Releases',
+      href: 'https://releases.hashicorp.com',
```

Comment:
Should this link to one of these two links?

- https://releases.hashicorp.com/waypoint/
- https://releases.hashicorp.com/waypoint-entrypoint/

Reply:
Honestly not sure - https://releases.hashicorp.com/waypoint/ certainly seems like a better bet than the current https://releases.hashicorp.com, will update with that! 👍